### PR TITLE
German Translation

### PR DESCRIPTION
**Pull request by krummrey**
## Description

Added german translation to translations.json and a Pokelist in german
## Motivation and Context

Not implemented yet
## How Has This Been Tested?

Runs on my site without problems
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by AL-dot-debug** [comment]
Thanks for trad! 
I'll merge it tonight as it looks that there's some conflict 

👍

File: core/json/translations.json [hunk 1]
```diff
@@ -1,492 +1,590 @@
...
+		"EN": "A PokémonGo Community in %s",
+		"FR": "Tout sur PokémonGo à %s",
+		"ES": "Una Comunidad PokémonGo en %s",
+		"DE": "Eine PokémonGo Community in %s"
+	},
+	"METADESCRIPTION": {
+		"EN": "Find all collected and proccessed data about PokémonGo in %s : nests, stats, frequency, ... Learn when and where to look for the Pokémon you're missing.",
+		"FR": "Trouvez toutes les informations et données sur PokémonGo à %s : les nids, les fréquences, les statistiques, ... Apprenez où et quand chercher le Pokémon qu'il vous manque !",
+		"ES": "Encuentra toda la información recolectada y procesada sobre PokémonGo en %s : nidos, estadísticas, frecuencia, ... Aprende cuándo y dónde buscar a los Pokémon que te faltan.",
+		"DE": "Find all collected and proccessed data about PokémonGo in %s : nests, stats, frequency, ... Learn when and where to look for the Pokémon you're missing."
```
Comment (NHGRisk) [inline core/json/translations.json]: translation missing

File: core/json/translations.json [hunk 2]
```diff
@@ -1,492 +1,590 @@
...
+		"EN": "unseen",
+		"FR": "absent",
+		"ES": "no vistos",
+		"DE": "Nicht gesehen"
+	},
+	"LURES": {
+		"EN": "Lures",
+		"FR": "Leurres",
+		"ES": "Cebos",
+		"DE": "Lock-Module"
```
Comment (NHGRisk) [inline core/json/translations.json:96]: "Lockmodule"

File: core/json/translations.json [hunk 3]
```diff
@@ -1,492 +1,590 @@
...
+		"EN": "Thanks and copyrights",
+		"FR": "Remerciements et copyrights",
+		"ES": "Agradecimientos y derechos",
+		"DE": "Danksagungen und Copyrights"
+	},
+	"FOOTER_SUB": {
+		"EN": "This website or the authors have no relationship with Niantic, Nintendo or other any Pokémon Go responsible.",
+		"FR": "Ce site ou ses auteurs ne sont en aucun cas en relation avec Niantic, Nintendo ou toute personne responsable de Pokémon Go.",
+		"ES": "Ni este sitio ni los autores tienen relación alguna con Niantic, Nintendo u otra entidad responsable por Pokémon Go.",
+		"DE": "This website or the authors have no relationship with Niantic, Nintendo or other any Pokémon Go responsible."
```
Comment (NHGRisk) [inline core/json/translations.json]: translation missing

File: core/json/translations.json [hunk 4]
```diff
@@ -1,492 +1,590 @@
...
+	},
+	"FOOTER_VISUAL_CONTENT": {
+		"EN": "<p>Icons designed by <a href='http://www.flaticon.com/packs/packs/pokemon-go/''>Roundicons Freebies</a> for <a href='http://www.flaticon.com'>Flaticon</a>. Font icons provided by <a href='http://fontawesome.io'>Font Awesome</a> by Dave Gandy.</p>",
+		"FR": "<p>Icônes créées par <a href='http://www.flaticon.com/packs/packs/pokemon-go/''>Roundicons Freebies</a> pour <a href='http://www.flaticon.com'>Flaticon</a>. Icon-Font fournie par <a href='http://fontawesome.io'>Font Awesome</a> de Dave Gandy.</p>",
+		"ES": "<p>Íconos creados por <a href='http://www.flaticon.com/packs/packs/pokemon-go/''>Roundicons Freebies</a> para <a href='http://www.flaticon.com'>Flaticon</a>. Íconos de fuentes provistos por <a href='http://fontawesome.io'>Font Awesome</a> de Dave Gandy.</p>",
+		"DE": "<p>Icons designed by <a href='http://www.flaticon.com/packs/packs/pokemon-go/''>Roundicons Freebies</a> for <a href='http://www.flaticon.com'>Flaticon</a>. Font icons provided by <a href='http://fontawesome.io'>Font Awesome</a> by Dave Gandy.</p>"
```
Comment (NHGRisk) [inline core/json/translations.json]: translation missing

File: core/json/translations.json [hunk 5]
```diff
@@ -1,492 +1,590 @@
...
-	   	"ES": "Nunca"
-   	},
-   	"TRAINER_GRAPH":{
-   		"EN": "Trainer <strong>level</strong> distribution",
-   		"FR": "<strong>Niveaux</strong> des entraîneurs Pokémon",
-   		"ES": "Distribución del <strong>nivel</strong> de entrenador Pokémon"
-   	},
+		"FR": "<p>Pokémon Go est ©2016 Niantic, Inc. ©2016 Pokémon. ©1995–2016 Nintendo / Creatures Inc. / GAME FREAK inc.</p><p>Pokémon et les personnages Pokémon ainsi que leurs noms sont des marques déposées par Nintendo.</p><p>Toutes les autres marques restent la propriété de leurs propriétaires légaux.</p>",
+		"ES": "<p>Pokémon Go es ©2016 Niantic, Inc. ©2016 Pokémon. ©1995–2016 Nintendo / Creatures Inc. / GAME FREAK inc.</p><p>Pokémon y los nombres de los personajes de Pokémon son marcas registradas por Nintendo.</p><p>Otras marcas están registradas por sus respectivos dueños.</p>",
+		"DE": "<p>Pokémon Go is ©2016 Niantic, Inc. ©2016 Pokémon. ©1995–2016 Nintendo / Creatures Inc. / GAME FREAK inc.</p><p>Pokémon and Pokémon character names are trademarks of Nintendo.</p><p>Other trademarks are the property of their respective owners.</p>"
```
Comment (NHGRisk) [inline core/json/translations.json:162]: translation missing

File: core/json/translations.json [hunk 6]
```diff
@@ -1,492 +1,590 @@
...
+		"EN": "Data collected since the <strong>2nd of August 2016</strong>",
+		"FR": "Données collectées depuis le <strong>2 Août 2016",
+		"ES": "Información recolectada desde el <strong>2 de Agosto de 2016",
+		"DE": "Daten seit dem <strong>06.09.2016</strong>"
+	},
+	"POKEDEX_TITLE": {
+		"EN": "The general <strong>Pokédex</strong><br><small>Gotta catch 'em all!</small>",
+		"FR": "Le <strong>Pokédex</strong><br><small>Attrapez les tous !</small>",
+		"ES": "<strong>Pokédex</strong> general<br><small>¡Atrápalos a todos!</small>",
+		"DE": "<strong>Pokédex</strong> Übersicht<br><small>Gotta catch 'em all!</small>"
```
Comment (NHGRisk) [inline core/json/translations.json:174]: translation missing

File: core/json/translations.json [hunk 7]
```diff
@@ -1,492 +1,590 @@
...
+		"EN": "The <strong>fight</strong> map<br><small>Follow the conquest of your team</small>",
+		"FR": "La carte des <strong>combats</strong><br><small>Suivez votre équipe !</small>",
+		"ES": "El mapa de <strong>combates</strong><br><small>¡Sigue las conquistas de tu equipo!</small>",
+		"DE": "Die <strong>Arena</strong> Map<br><small>Folge dem Erfolg deines Teams!</small>"
+	},
+	"GYMS_METATITLE": {
+		"EN": "Follow the conquest",
+		"FR": "Suivez la conquête",
+		"ES": "Sigue la conquista",
+		"DE": "Follow the conquest"
```
Comment (NHGRisk) [inline core/json/translations.json:198]: translation missing

File: core/json/translations.json [hunk 8]
```diff
@@ -1,492 +1,590 @@
...
+	"NAV_DASHBOARD": {
+		"EN": "Dashboard",
+		"FR": "Tableau de bord",
+		"DE": "Dashboard"
+	},
+	"GYMS_METADESC": {
+		"EN": "Valor, Mystic or Instinct? Who will win the fight?",
+		"FR": "Bravoure, Sagesse ou Intuition ? Qui dominera ?",
+		"ES": "¿Valor, Místico o Instinto ? ¿Quién ganará?",
+		"DE": "Valor, Mystic or Instinct? Who will win the fight?"
```
Comment (NHGRisk) [inline core/json/translations.json]: translation missing

File: core/json/translations.json [hunk 9]
```diff
@@ -1,492 +1,590 @@
...
+		"EN": "Stats",
+		"FR": "Caractéristiques",
+		"ES": "Características",
+		"DE": "Statistiken"
+	},
+	"POKEMON_FAMILY": {
+		"EN": "Family",
+		"FR": "Famille",
+		"ES": "Familia",
+		"DE": "Famile"
```
Comment (NHGRisk) [inline core/json/translations.json:329]: "Familie"

File: core/json/translations.json [hunk 10]
```diff
@@ -1,492 +1,590 @@
...
+	},
 	"DASHBOARD_TITLE":{
-   		"EN": "MOAR <strong>DATA</strong> MOAAAAR",
-   		"FR": "ENCORE DES <strong>DONNÉES</strong> ENCOOORE",
-   		"ES": "Más <strong>DATOS</strong> Más"
-   	},
+		"EN": "MOAR <strong>DATA</strong> MOAAAAR",
+		"FR": "ENCORE DES <strong>DONNÉES</strong> ENCOOORE",
+		"ES": "Más <strong>DATOS</strong> Más",
+		"DE": "MOAR <strong>DATA</strong> MOAAAAR"
```
Comment (NHGRisk) [inline core/json/translations.json:455]: translation missing

File: core/json/translations.json [hunk 11]
```diff
@@ -1,492 +1,590 @@
...
+	},
 	"DASHBOARD_LURES":{
-   		"EN": "Active lures",
-   		"FR": "Leurres actifs",
-   		"ES": "Cebos activos"
-   	},
+		"EN": "Active lures",
+		"FR": "Leurres actifs",
+		"ES": "Cebos activos",
+		"DE": "Aktive Lock-Module"
```
Comment (NHGRisk) [inline core/json/translations.json:539]: "Aktive Lockmodule"

File: core/json/translations.json [hunk 12]
```diff
@@ -1,492 +1,590 @@
...
+	},
 	"DASHBOARD_GRAPH_MYSTIC_PRESTIGE_AVERAGE":{
-   		"EN": "Mystic Prestige Average",
-   		"FR": "Sagesse - Prestige Moyen",
-   		"ES": "Místico - Medio de lujo"
-   	},
+		"EN": "Mystic Prestige Average",
+		"FR": "Sagesse - Prestige Moyen",
+		"ES": "Místico - Medio de lujo",
+		"DE": "Mystic Prestige Average"
```
Comment (NHGRisk) [inline core/json/translations.json]: translation missing

File: core/json/translations.json [hunk 13]
```diff
@@ -1,492 +1,590 @@
...
+	},
 	"DASHBOARD_GRAPH_VALOR_PRESTIGE_AVERAGE":{
-   		"EN": "Valor Prestige Average",
-   		"FR": "Bravoure - Prestige Moyen",
-   		"ES": "Valor - Medio de lujo"
-   	},
+		"EN": "Valor Prestige Average",
+		"FR": "Bravoure - Prestige Moyen",
+		"ES": "Valor - Medio de lujo",
+		"DE": "Valor Prestige Average"
```
Comment (NHGRisk) [inline core/json/translations.json]: translation missing

File: core/json/translations.json [hunk 14]
```diff
@@ -1,492 +1,590 @@
...
+	},
 	"DASHBOARD_GRAPH_INSTINCT_PRESTIGE_AVERAGE":{
-   		"EN": "Instinct Prestige Average",
-   		"FR": "Intuition - Prestige Moyen",
-   		"ES": "Instinto - Medio de lujo"
-   	},
+		"EN": "Instinct Prestige Average",
+		"FR": "Intuition - Prestige Moyen",
+		"ES": "Instinto - Medio de lujo",
+		"DE": "Instinct Prestige Average"
```
Comment (NHGRisk) [inline core/json/translations.json]: translation missing

File: core/json/translations.json [hunk 15]
```diff
@@ -1,492 +1,590 @@
...
+	},
 	"DASHBOARD_GRAPH_MYSTIC_GYM_OWNED":{
-   		"EN": "Mystic Gym Owned",
-   		"FR": "Sagesse - Arènes conquises",
-   		"ES": "Místico - Arenas conquistado"
-   	},
+		"EN": "Mystic Gym Owned",
+		"FR": "Sagesse - Arènes conquises",
+		"ES": "Místico - Arenas conquistado",
+		"DE": "Mystic Gym Owned"
```
Comment (NHGRisk) [inline core/json/translations.json]: translation missing

File: core/json/translations.json [hunk 16]
```diff
@@ -1,492 +1,590 @@
...
+	},
 	"DASHBOARD_GRAPH_VALOR_GYM_OWNED":{
-   		"EN": "Valor Gym Owned",
-   		"FR": "Bravoure - Arènes conquises",
-   		"ES": "Valor - Arenas conquistado"
-   	},
+		"EN": "Valor Gym Owned",
+		"FR": "Bravoure - Arènes conquises",
+		"ES": "Valor - Arenas conquistado",
+		"DE": "Valor Gym Owned"
```
Comment (NHGRisk) [inline core/json/translations.json]: translation missing

File: core/json/translations.json [hunk 17]
```diff
@@ -1,492 +1,590 @@
...
+	},
 	"DASHBOARD_GRAPH_INSTINCT_GYM_OWNED":{
-   		"EN": "Instinct Gym Owned",
-   		"FR": "Intuition - Arènes conquises",
-   		"ES": "Instinto - Arenas conquistado"
-   	},
+		"EN": "Instinct Gym Owned",
+		"FR": "Intuition - Arènes conquises",
+		"ES": "Instinto - Arenas conquistado",
+		"DE": "Instinct Gym Owned"
```
Comment (NHGRisk) [inline core/json/translations.json]: translation missing

File: core/json/translations.json [hunk 18]
```diff
@@ -1,492 +1,590 @@
...
-      "EN": "Page <strong>not found</strong><br><small>There must have been some kind of mistake.</small>",
-      "FR": "Page <strong>introuvable</strong><br/><small>Il y a du y avoir une erreur</small>",
-      "NL": "Pagina <strong>niet gevonden</strong><br/><small>&nsbp;</small>",
-      "ES": "Página <strong>no encontrada</strong><br/><small>&nsbp;</small>"
-    }
- }
+		"EN": "Lured Pokestop",
+		"FR": "Pokéstops avec leurre",
+		"ES": "Poképaradas con cebos",
+		"DE": "Lured Pokestop"
```
Comment (NHGRisk) [inline core/json/translations.json]: translation missing

File: core/json/translations.json [hunk 19]
```diff
@@ -1,492 +1,590 @@
...
+		"EN": "Valor",
+		"FR": "Bravoure",
+		"ES": "Valor",
+		"DE": "Mystic"
+	},
+	"INSTINCT": {
+		"EN": "Instinct",
+		"FR": "Intuition",
+		"ES": "Instinto",
+		"DE": "Instinkt"
```
Comment (NHGRisk) [inline core/json/translations.json:72]: "Intuition"

File: core/json/translations.json [hunk 20]
```diff
@@ -1,492 +1,590 @@
...
+		"EN": "Mystic",
+		"FR": "Sagesse",
+		"ES": "Místico",
+		"DE": "Mystic"
+	},
+	"VALOR": {
+		"EN": "Valor",
+		"FR": "Bravoure",
+		"ES": "Valor",
+		"DE": "Mystic"
```
Comment (NHGRisk) [inline core/json/translations.json]: Wagemut

File: core/json/translations.json [hunk 21]
```diff
@@ -1,492 +1,590 @@
...
+		"EN": "Find ",
+		"FR": "Trouvez ",
+		"ES": "Buscar ",
+		"DE": "Finde "
+	},
+	"MYSTIC": {
+		"EN": "Mystic",
+		"FR": "Sagesse",
+		"ES": "Místico",
+		"DE": "Mystic"
```
Comment (NHGRisk) [inline core/json/translations.json]: Weisheit

**Comment by NHGRisk** [comment]
If you don't wanna correct the "mistakes" i can do it this evening
danke für die arbeit dennoch!

File: core/json/translations.json [hunk 22]
```diff
@@ -542,43 +542,43 @@
...
 		"ES": "Valor - Medio de lujo",
-		"DE": "Valor Prestige Average"
+		"DE": "Wagemut - Prestige Durchschnitt"
 	},
 	"DASHBOARD_GRAPH_INSTINCT_PRESTIGE_AVERAGE":{
 		"EN": "Instinct Prestige Average",
 		"FR": "Intuition - Prestige Moyen",
 		"ES": "Instinto - Medio de lujo",
-		"DE": "Instinct Prestige Average"
+		"DE": "Instinkt - Prestige Durchschnitt"
```
Comment (NHGRisk) [inline core/json/translations.json:557]: Intuition

File: core/json/translations.json [hunk 23]
```diff
@@ -542,43 +542,43 @@
...
 		"ES": "Valor - Arenas conquistado",
-		"DE": "Valor Gym Owned"
+		"DE": "Arenen von Team Wagemut"
 	},
 	"DASHBOARD_GRAPH_INSTINCT_GYM_OWNED":{
 		"EN": "Instinct Gym Owned",
 		"FR": "Intuition - Arènes conquises",
 		"ES": "Instinto - Arenas conquistado",
-		"DE": "Instinct Gym Owned"
+		"DE": "Arenen von Tem Instinkt"
```
Comment (NHGRisk) [inline core/json/translations.json]: Team Intuition

**Comment by Obihoernchen** [comment]
Please add `de` to `core/json/variables.examples.json` as well :)

**Comment by krummrey** [comment]
I made a bunch of changes in my repo. Will they show up here or do I have to do something so that this request is updated?

**Comment by NHGRisk** [comment]
@krummrey i think u gotta update it manually

**Comment by Obihoernchen** [comment]
@krummrey they show up automatically. You don't have to do anything except pushing the changes to your repo. 
Please git pull the latest code from original repo and resolve merge conflicts 👍

**Comment by brusselopole** [comment]
This branch has conflicts that must be resolved

@NHGRisk could you check please?

**Review by Obihoernchen — CHANGES_REQUESTED** [review]
Do not merge yet! 

There are still issues with some translations. 
Please add NHGRisks comments. But then there are still some issues left. For example it must be team "Intuition".

**Comment by NHGRisk** [comment]
I updated & corrected it https://github.com/brusselopole/Worldopole/pull/68

**Comment by Obihoernchen** [comment]
Let's use #68 cause it has no merge conflicts and more fixes. 
Thank you @krummrey for the base!

**Pull request closed without merging**